packages/server/src/auth/dto/auth.dto.ts

```diff
@@ -67,7 +74,13 @@ export class SignupTransformPipe implements PipeTransform {
 export class UsernameLoginTransformPipe implements PipeTransform {
   transform(body: any): UsernameLoginDto {
     const user = new UserSignupDto();
-    user.project_id = body.project_id.toString();
+    const properties = ['projectId', 'username', 'password']
```

Comment:
I believe NestJS's validation handles all of this already

Reply:
Originally I has `new ValidationPipe()` follow my custom `new TransformationPipe()`, but it was rejecting everything. I'll look into `ValidationPipe` again.

Reply:
Ah ok. So I removed `method` in the `UserSignupDto` definition since it's never sent in the user body. That was why the validation was rejecting it. Or do we want to keep this and the user has to specify the method. I think it would make more sense that the code detects which one was given and handle it from there.